darts/tests/dataprocessing/transformers/test_window_transformations.py

```diff
@@ -494,12 +544,18 @@ def test_transformers_pipeline(self):
         expected_transformed_series = TimeSeries.from_times_and_values(
             times1,
             [100, 15, 30, 45, 60, 75, 90, 105, 120, 135],
-            columns=["rolling_sum_3_2_0"],
+            columns=["rolling_TotalOperation_3_2_0"],
         )
 
         # adds NaNs
         window_transformations = [
-            {"function": "sum", "mode": "rolling", "window": 3, "min_periods": 2}
+            {
+                "function": "sum",
+                "function_name": "TotalOperation",
```

Comment:
I would not use the "function_name" feature in this test, just to make sure that the default behavior when `fn` is not "apply" works correctly.

Reply:
Sounds good to me. My original intention is to test that for initialization via `WindowTransformer`, we could also get a customized name correctly and not want to initialize a new test to simply check the name in a repeated manner. I will remove this test